### ISSUE TITLE
P3-Project_Planning

### ISSUE THREAD

**Issue by jameslyleyoung** [cfiller123/liftoff-assignments#2]
Wireframes are clean and precise, I would say Satisfactory.  Will you be creating more to help you 'design' your project? As I see that you have some code in your repo, and it looks like you are moving right along with your project.  You don't need anymore for the grade, I was just curious if you were planning on using Wireframes to help you build out the rest of your features.

**Comment by cfiller123** [comment]
Yeah, I think so - I'll update this when I update some more wireframes.
